airflow/www_rbac/templates/airflow/couture_config.html

```diff
@@ -322,64 +239,69 @@ <h3>Configurations</h3>
+      +'<br>'
     + '</div>'
-    + '<div class="couturecol-75">'
-    + '<input name="new-arg-value-' + globalValues.newArgKeySN + '" type="text" value="" placeholder="Enter value">'
+    + '<div class="col-xs-8">'
+    +   '<input class="form-control" name="new-arg-value-' + globalValues.newArgKeySN + '" type="text" value="" placeholder="Enter value">'
     + '</div>';
   '</div>'
   div1.innerHTML = template;
   globalValues.newArgKeySN += 1;
   document.getElementById('container1').appendChild(div1);
 }
 
+//function to add another row of fields for configurations inputs
 function add_feed_config()
 {
   var div1 = document.createElement('div');
-
-  // creating template here.
-  const template = '<div class="couturerow">'
-    + '<div class="couturecol-20">'
-      + '<input name="new-config-key-'+ globalValues.newConfigKeySN + '" type="text" value="" placeholder="Enter key">'
+  const template = '<div class="row">'
+    + '<div class="col-xs-3">'
+      + '<input class="form-control" name="new-config-key-'+ globalValues.newConfigKeySN + '" type="text" value="" placeholder="Enter key">'
+      +'<br>'
     + '</div>'
-    + '<div class="couturecol-75">'
-    + '<input name="new-config-value-' + globalValues.newConfigKeySN + '" type="text" value="" placeholder="Enter value">'
+    
+    + '<div class="col-xs-8">'
+    + '<input class="form-control" name="new-config-value-' + globalValues.newConfigKeySN + '" type="text" value="" placeholder="Enter value">'
     + '</div>';
   '</div>'
   div1.innerHTML = template;
   globalValues.newConfigKeySN += 1;
   document.getElementById('container2').appendChild(div1);
 }
 
-function del_feed(option)
+//deletes an entire row of the field for args or configs as specified by user
+function del_feed_test(option, element)
 {
   let del = confirm("Are you sure you want to delete?");
-  if(del) {
-    document.getElementById("for_delete").value = option;
-    var div1 = document.createElement('div');
-    div1.innerHTML = document.getElementById('newfield_for_delete').innerHTML;
-    document.getElementById('container1').appendChild(div1);
+  let update = 'newfield_for_delete';
+  let containerNo = 'container1';
+  if(element == 'for_delete_config'){
+    update += '_config';
+    containerNo = 'container2';
   }
-}
-function del_feed_config(option)
-{
-  let del = confirm("Are you sure you want to delete?");
   if(del) {
-    document.getElementById("for_delete_config").value = option;
+    document.getElementById(element).value = option;
     var div1 = document.createElement('div');
-    div1.innerHTML = document.getElementById('newfield_for_delete_config').innerHTML;
-    document.getElementById('container2').appendChild(div1);
+    div1.innerHTML = document.getElementById(update).innerHTML;
+    document.getElementById(containerNo).appendChild(div1);
   }
 }
+
 var expanded = false;
-count_of_clicks=1
+count_of_clicks=1;
+var Py_expanded = false;
```

Comment:
change the function name. Also, either use camelCase or snake_case. Don't use both. For function names, use camelCase.

Reply:
Change the naming everywhere where it is not proper.